app/client/style.css

```diff
@@ -2,3 +2,11 @@
   background: url('images/spiral-of-stars.jpeg') no-repeat center center fixed;
   background-size: cover;
 }
+
+#blue-a-hyperlink {
```

Comment:
maybe this should be a class instead or be baked directly into the style for the component.

Reply:
converted id to inline style